pil_analyzer/src/pil_analyzer.rs

```diff
@@ -191,7 +191,7 @@ impl<T: FieldElement> PILContext<T> {
             PilStatement::ConstantDefinition(_, name, value) => {
                 self.handle_constant_definition(name, value)
             }
-            PilStatement::LetStatement(start, name, None) => {
+            PilStatement::LetStatement(start, name, _type_name, None) => {
```

Comment:
Maybe better to use just '_'? I remember there is a difference (but I am sure the optimizer makes it irrelevant).